spectrocloud/resource_cluster_group.go

```diff
@@ -88,6 +88,14 @@ func resourceClusterGroup() *schema.Resource {
 							Optional: true,
 							Default:  "",
 						},
+						"kubernetes_distro": {
```

Comment:
May be rename to "k8s_distribution" ? @SivaanandM

Reply:
done